README.md

```diff
@@ -44,22 +67,31 @@ flutter build macos
 Builds for Least Authority servers:
 
 ```bash
-flutter build linux lib/main_la.dart
+flutter build linux -t lib/main_la.dart
 ```
 
 Builds for local instances:
 
 ```bash
-flutter build linux lib/main_local.dart
+flutter build linux -t lib/main_local.dart
 ```
 
+More information on signing releases can be found [here](https://github.com/LeastAuthority/destiny/blob/main/doc/releases.md)
+
 ### Starting the wormhole services locally
 
 ```bash
 cd docker
 docker-compose up -d
 ```
 
- Known Issues:
-Flutter currently [does not support x86 for android builds](https://github.com/flutter/flutter/issues/9253),
+## Known Issues:
+- Flutter currently [does not support x86 for android builds](https://github.com/flutter/flutter/issues/9253),
 so x86 emulators or devices are not supported.
+- MacOS M1 chip build is not supported.
+
+## Other
+
+- **FAQ** can be found [here](https://github.com/LeastAuthority/destiny/blob/main/FAQ.md).
+- **Private Policy** can be found [here](https://github.com/LeastAuthority/destiny/blob/main/PRIVACY-POLICY.md).
+- **Terms & Conditions** can be found [here](https://github.com/LeastAuthority/destiny/blob/main/TERMS.md.
```

Comment:
miss parenthesis at the end of the terms link so the hyperlink is not working

Reply:
fixed